django/db/migrations/serializer.py

```diff
@@ -270,6 +272,40 @@ def serialize(self):
         return "uuid.%s" % repr(self.value), {"import uuid"}
 
 
+class ClassSerializer(BaseSerializer):
+    def serialize(self):
+        _module = self.value.__module__
+        module_first_part, module_second_part = _module.split('.')
+        _class = self.value.__class__
+        _path = ""
+        if module_second_part.startswith('_'):
+            related_modules = [
+                m for m in sys.modules
+                if module_first_part in m and
+                m != _module and len(m.split('.')) > 1 and not
+                m.split('.')[1].startswith('_')
+            ]
+            for related_module in related_modules:
+                klass_members = dict(
+                    inspect.getmembers(
+                        sys.modules[related_module],
+                        lambda m: inspect.isclass(m))).keys()
+                if _class.__name__ not in klass_members:
+                    continue
+                _path = related_module + "." + _class.__name__
+                break
```

Comment:
This is not performing any identity check so if a related module happens to have a class the same name (it's not that uncommon to have classes with the same name namespaced in different modules) then it would be considered to be the same class.